test/test_dht.cpp

```diff
@@ -3305,7 +3316,7 @@ TORRENT_TEST(invalid_error_msg)
 	node.incoming(node.m_sock, m);
 
 	bool found = false;
-	for (int i = 0; i < int(observer.m_log.size()); ++i)
+	for (std::size_t i = 0; i < observer.m_log.size(); ++i)
```

Comment:
this looks like it could be a range-for loop over ``observer.m_log``.

Reply:
done